asgal

```diff
@@ -351,69 +351,80 @@ def runASGAL(args, genes, chr_genes_dict):
         ref = args.refPath
         anno = args.annoPath
         samples = [args.sample1Path]
+
+        # paired-end read
+        if args.sample2Path != '-' and not args.multiMode:
+            samples += [args.sample2Path] # samples has length 2 [read1.fa, read2.fa]
+
```

Comment:
Is this condition correct?

Reply:
Since you are in the else branch, you are sure that `args.multiMode` is `False` so I think it's useless to check it again